modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_headed_text.yml

```diff
@@ -0,0 +1,35 @@
+id: az_paragraph_headed_text
+label: AZ Paragraph Headed Text
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: az_paragraphs_item
+  bundle: uaqs_headed_text
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_text
+
+process:
+
+  field_az_text_area:
+    plugin: sub_process
+    source: field_uaqs_summary
+    process:
+      delta: delta
+      value: value
+      summary: summary
```

Comment:
The `summary` field is not available on `field_az_text_area`

Reply:
`summary` field is removed.